README.md

```
@@ -135,10 +179,24 @@ Contributions, suggestions, bug reports and fixes are welcome!
 
 For new features, components, or extensions, please open an issue and discuss before sending a PR.
 
+If you'd like to help translate LibreChat into your language, we'd love your contribution! Improving our translations not only makes LibreChat more accessible to users around the world but also enhances the overall user experience. Please check out our [Translation Guide](.github/TRANSLATION.md).
+
 ---
 
 ## 💖 This project exists in its current state thanks to all the people who contribute
 
 <a href="https://github.com/danny-avila/LibreChat/graphs/contributors">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: a

(MD033, no-inline-html)